app/controllers/users_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
   end
 
   def index
-    @pagy, @users = pagy(@users,  items: Company::PAGE_SIZE)
+    @pagy, @users = pagy(@users)
```

Comment:
It is users_controller so Company::PAGE_SIZE is a bit weird, besides we have a default set in pagy.rb so i removed it